pkg/component/kubernetes/scheduler/scheduler.go

```diff
@@ -115,9 +117,60 @@ type kubeScheduler struct {
 	image          string
 	replicas       int32
 	config         *gardencorev1beta1.KubeSchedulerConfig
+	values         Values
+}
+
+type Values struct {
+	// ClusterType is the type of the cluster.
+	ClusterType component.ClusterType
 }
```

Comment:
`kube-scheduler` is deployed only in the controlplane (seed). No need of ClusterType here.